nucleus/admin/server-mgmt/src/main/java/com/sun/enterprise/admin/servermgmt/cli/ChangeMasterPasswordCommand.java

```diff
@@ -136,13 +136,15 @@ protected int executeCommand() throws CommandException {
                 arguments.add(getDefaultNodesDirs().getAbsolutePath());
                 arguments.add(domainNameOrNodeName);
                 String[] newargs = (String[]) arguments.toArray(new String[arguments.size()]);
-
+                
                 command = CLICommand.getCommand(habitat,
-                        CHANGE_MASTER_PASSWORD_NODE);
+                CHANGE_MASTER_PASSWORD_NODE);
                 return command.execute(newargs);
             }
         } catch (IOException e) {
             throw new CommandException(e.getMessage(),e);
+        } finally {
+            System.out.println("WARNING: Additional Truststores are not re-encrypted, it is your responsibility to do this.");
```

Comment:
I wouldn't use System.out - its use is generally discouraged.

Use `Logger.getLogger(CLICommand.class.getPackage().getName());`